test/CI/Travis/setup_selenium.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# set up Selenium for functional tests
+
+wget http://goo.gl/cvntq5 -O selenium.jar
```

Comment:
this is the question I mentioned in the main issue: which version of selenium to test against? always the latest one, or a fixed one? or maybe one different selenium version for each php version?